drivers/entropy/Kconfig.nrf5

```diff
@@ -61,10 +76,10 @@ config ENTROPY_NRF5_ISR_THRESHOLD
 	  buffer goes below this number hardware entropy generation will be
 	  started.
 
+endif #ENTROPY_NRF5_RNG_DEFAULT
 
 config ENTROPY_NRF5_PRI
 	int "RNG interrupt priority"
-	depends on ENTROPY_NRF5_RNG
```

Comment:
I think this dependency should not be removed. Since driver will never be used if ENTROPY_NRF5_RNG is not selected.

Reply:
It is under if ENTROPY_NRF5_RNG/endif, so the dependency is redundant